.gitignore

```diff
@@ -1 +1,28 @@
+# NodeJS projects
+!node_modules
+node_modules/*
+
+node_modules/@asu-design-system
+!node_modules/@asu-design-system
+node_modules/@asu-design-system/*
+!node_modules/@asu-design-system/component-events
+node_modules/@asu-design-system/component-events/*
+!node_modules/@asu-design-system/component-events/dist
+node_modules/@asu-design-system/component-events/dist/*
+!node_modules/@asu-design-system/component-events/dist/events.production.js
+!node_modules/@asu-design-system/component-events/dist/vendor.production.js
+!node_modules/@asu-design-system/component-events/dist/asuEvents.umd.js
+!node_modules/@asu-design-system/component-events/dist/vendor.umd.js
+
```

Comment:
There is no node_modules component_events package in this module.

Reply:
Removed all references to @asu-design-system (no JS in the module)